woocommerce/woocommerce-gutenberg-products-block
# Fix: Ensure new filter blocks work with inherited query context

## What

Fixes #11620 

This PR:
- removes the ancestor property from the `Collection Filters` block metadata to make the block available globally.
- ensures the block work when added outside the `Product Collection` block.
- removes the editor contextual.

## Why

- We need to make sure the new filters blocks preserve the behavior of the current ones.
- Allowing blocks to be added outside the `Product Collection` block make the new blocks more flexible and avoid confusion. The block will inherit the closest context
- Removing the editor contextual makes our block codebase smaller and simpler. Having editor contextual doesn't add any notable benefit to end users as the filter data is dynamic and can be changed anytime.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Go to a page or template.
2. Open the global inserter, search for Collection Filters block.
3. See it available in the inserter as well as can be added to the page as a top level block.
4. See new blocks still work in editor and frontend as before.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
